### DISCUSSION TITLE
[Discussion] - Holding Monika, what do they feel?

### DISCUSSION THREAD

**Discussion by WingdingsMaster** [Monika-After-Story/MonikaModDev#7910] [general]
What I think Monika feels is a invisible force that they know is the player, and it feels like a normal hug, maybe that's why they close there eyes? is there a actual answer?

**Comment by Freshairkaboom** [comment]
I think she's imagining it, there doesn't seem to be any invisible force in the script that I'm aware of. It's kind of like when a family member send you virtual hugs, and you close your eyes for a moment to receive them.

Just me?
